### DISCUSSION TITLE
Organization of code

### DISCUSSION THREAD

**Discussion by onx2** [Sixze/ALS-Refactored#498] [Ideas]
What are your thoughts on using things like `#pragma region Some Label` in C++? I think it would make the codebase easier to read / navigate. Here is an example of what it might look like -- each region is collapsible and I've used big comment as just a matter of style / I think it is easier to scan things

![image](https://github.com/Sixze/ALS-Refactored/assets/23039725/c4ab9628-f406-4d98-8a11-289fe7f770ea)

**Comment by ameaninglessname** [comment]
IMO, the code will be divided into two part : data member and function member, in order.

In each part, `#pragma region` could be added.

(Which looks like what the code of `ALS-Refactored` does right now.)
